lib/model/maalcie/MaaltijdAbonnementenModel.class.php

```diff
@@ -220,67 +199,32 @@ public static function inschakelenAbonnementVoorNovieten($mrid) {
 	 * @return MaaltijdAbonnement[]|int OR aantal nieuwe abonnementen novieten
 	 * @throws Exception
 	 */
-	private static function newAbonnement($mrid, $uid = null) {
```

Comment:
Deze is ook voor de "Geef novieten een abo op"-knop.

Alle $var = null hebben zo'n fancy feature. Geef daar extra aandacht aan bij het refactoren en testen, dan gaat het vast goed.